de4dot.code/deobfuscators/dotNET_Reactor/v4/EncryptedResource.cs

```diff
@@ -208,7 +208,8 @@ class DecrypterV1 : IDecrypter {
 				requiredTypes.AddRange(additionalTypes);
 				if (!localTypes.All(requiredTypes))
 					return false;
-
+				if (localTypes.Exists("System.UInt32")) // DNR >=4.8
```

Comment:
I haven't had time to test it, but will it work if you move this test inside the 'if' below? I think that's the path new versions will take

Reply:
Yes, it still works with the test moved into other if. I agree with you, it makes sense that new versions have a UInt64 or a UInt32 and that we should only test for this if there is a `SymmetricAlgorithm` present. I've commited the new changes.